test/vaadin-overlay.html

```diff
@@ -292,6 +292,81 @@
           // (in this case overlay 1 at step 3)
           expect(getComputedStyle(overlay1Part).pointerEvents).to.equal('auto');
         });
+
+        describe('modeless', () => {
+
+          let modeless1, modeless2;
+
+          const getFrontmostOverlayFromScreenCenter = () => {
+            let elementFromPoint = document.elementFromPoint(window.innerWidth / 2, window.innerHeight / 2);
+            while (elementFromPoint && elementFromPoint.localName !== 'vaadin-overlay') {
+              elementFromPoint = elementFromPoint.host || elementFromPoint.parentNode;
+            }
+            return elementFromPoint;
+          };
+
+          beforeEach(() => {
+            Array.from(document.body.children).forEach(element => element.opened = false);
+            modeless1 = fixture('default').firstElementChild.firstElementChild;
+            modeless2 = fixture('default').firstElementChild.firstElementChild;
+            modeless1.modeless = modeless2.modeless = true;
+          });
```

Comment:
There are test failures probably caused by stacking of the overlays.
We could add:
```
afterEach(() => {
  modeless1.opened = false;
  modeless2.opened = false;
});
``` to avoid those.

Reply:
Done